README.md

```diff
@@ -93,21 +93,28 @@ MariaDB is not tested/recommended.
 **Note:** SQLite is used in Airflow tests. Do not use it in production. We recommend
 using the latest stable version of SQLite for local development.
 
-## Support for Python versions
+## Support for Python and Kubernetes versions
 
-As of Airflow 2.0 we agreed to certain rules we follow for Python support. They are based on the official
-release schedule of Python, nicely summarized in the
-[Python Developer's Guide](https://devguide.python.org/#status-of-python-branches)
+As of Airflow 2.0 we agreed to certain rules we follow for Python and Kubernetes support.
+They are based on the official release schedule of Python and Kubernetes, nicely summarized in the
+[Python Developer's Guide](https://devguide.python.org/#status-of-python-branches) and
+[Kubernetes version skew policy](https://kubernetes.io/docs/setup/release/version-skew-policy/).
 
-1. We finish support for Python versions when they reach EOL (For Python 3.6 it means that we will remove it
-   from being supported on 23.12.2021).
+1. We drop support for Python and Kubernetes versions when they reach EOL. We drop support for those
+   EOL versions in master right after EOL data, and it is effectively removed when we release the
+   first new MINOR (Or MAJOR if there is no new MINOR version) of Airflow
+   For example for Python 3.6 it means that we drop support in master right after 23.12.2021, and the first
+   MAJOR or MINOR version of Airflow released after will not have it.
 
-2. The "oldest" supported version of Python is the default one. "Default" is only meaningful in terms of
-   "smoke tests" in CI PRs which are run using this default version.
+2. The "oldest" supported version of Python/Kubernetes is the default one. "Default" is only meaningful
+   in terms of "smoke tests" in CI PRs which are run using this default version and default reference
+   image available in DockerHub. Currently ``apache/airflow`` and ``apache/airflow:2.0.2`` images
```

Comment:
```suggestion
   image available in DockerHub. Currently ``apache/airflow:2.0.2-python3.6`` and ``apache/airflow:2.0.2`` images
```

Did you mean this?

Reply:
No this was correct. We also have `apache/airflow` image (`apache/airflow:latest`)

Reply:
```
docker run -it apache/airflow version                
2.0.2
```

Reply:
Well the `latest` image was not updated yesterday :D and the following would be more clearer I think

```suggestion
   image available in DockerHub. Currently ``apache/airflow:latest`` and ``apache/airflow:2.0.2`` images
```

Reply:
Indeed. I cheated a bit ;). I will update the release docs to include pushing the image . Added :latest to the docs as well.